tests/ImageSharp.Tests/Drawing/Text/DrawTextOnImageTests.cs

```diff
@@ -65,6 +65,24 @@ public void DoesntThrowExceptionWhenOverlappingRightEdge_Issue688<TPixel>(TestIm
             }
         }
 
+        [Theory]
+        [WithSolidFilledImages(1500, 500, "White", PixelTypes.Rgba32)]
+        public void DoesntThrowExceptionWhenOverlappingRightEdge_Issue688_2<TPixel>(TestImageProvider<TPixel> provider)
```

Comment:
There is a test above this which is very similar and was targeted to hit the same bug but based on the comments of #688, it doesn't actually hit the issue (or at least not the same way). With this in mind, the names of the tests are extremely similar (mine has "_2") so I'm not sure if I should just replace that test with this one or give mine a different name.